Server/src/middleware/JWTAuthenticator.ts

```diff
@@ -31,7 +31,15 @@ export class JWTAuthenticator {
 
     static verifyAdmin(request: Request, response: Response, next: NextFunction): Response {
         // Lets us protect admin only backend routes vs any user routes
-        if (request.body.user.account_admin !== 1) {
+        if (!((request.body.user.account_admin == 1) || (request.body.user.account_admin == 2))) {
```

Comment:
imo at this point just do if(request.body.user.account_admin == 1 || ... == 2) {
next()
} else {
return 403
}
the nested condition is hard to read

Reply:
Good suggestion, added